crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -225,13 +210,12 @@ impl DataTransfer {
         threshold: usize,
     ) -> Result<(), ParquetError> {
         // The clone is performed to not create a deadlock with transfer_data().
-        for table_name_column_index_size_in_bytes in self.compressed_files.clone().iter() {
-            let table_name = &table_name_column_index_size_in_bytes.key().0;
-            let column_index = table_name_column_index_size_in_bytes.key().1;
+        for table_name_column_index_size_in_bytes in self.table_size_in_bytes.clone().iter() {
```

Comment:
Maybe change to something like `for table_name_size_in_bytes in`.

Reply:
Fixed.